infra-as-code/bicep/modules/managementGroups/managementGroups.bicep

```
@@ -159,52 +163,27 @@ resource resDecommissionedMg 'Microsoft.Management/managementGroups@2021-04-01'
   }
 }
 
-// Level 3 - Child Management Groups under Platform MG
-resource resPlatformManagementMg 'Microsoft.Management/managementGroups@2021-04-01' = {
-  name: varPlatformManagementMg.name
-  properties: {
-    displayName: varPlatformManagementMg.displayName
-    details: {
-      parent: {
-        id: resPlatformMg.id
-      }
-    }
-  }
-}
-
-resource resPlatformConnectivityMg 'Microsoft.Management/managementGroups@2021-04-01' = {
-  name: varPlatformConnectivityMg.name
-  properties: {
-    displayName: varPlatformConnectivityMg.displayName
-    details: {
-      parent: {
-        id: resPlatformMg.id
-      }
-    }
-  }
-}
-
-resource resPlatformIdentityMg 'Microsoft.Management/managementGroups@2021-04-01' = {
-  name: varPlatformIdentityMg.name
+// Level 3 - Child Management Groups under Landing Zones MG
+resource resLandingZonesChildMgs 'Microsoft.Management/managementGroups@2021-04-01' = [for mg in items(varLandingZoneMgChildrenUnioned): if (!empty(varLandingZoneMgChildrenUnioned)) {
+  name: '${parTopLevelManagementGroupPrefix}-landingzones-${mg.key}${parTopLevelManagementGroupSuffix}'
   properties: {
-    displayName: varPlatformIdentityMg.displayName
+    displayName: mg.value.displayName
     details: {
       parent: {
-        id: resPlatformMg.id
+        id: resLandingZonesMg.id
       }
     }
   }
-}
-
-// Level 3 - Child Management Groups under Landing Zones MG
+}]
 
-resource resLandingZonesChildMgs 'Microsoft.Management/managementGroups@2021-04-01' = [for mg in items(varLandingZoneMgChildrenUnioned): if (!empty(varLandingZoneMgChildrenUnioned)) {
-  name: '${parTopLevelManagementGroupPrefix}-landingzones-${mg.key}${parTopLevelManagementGroupSuffix}'
+//Level 3 - Child Management Groups under Platform MG
+resource resPlatformChildMgs 'Microsoft.Management/managementGroups@2021-04-01' = [for mg in items(varPlatformMgChildrenUnioned): if (!empty(varPlatformMgChildrenUnioned)) {
+  name: '${parTopLevelManagementGroupPrefix}-platform-${mg.key}'
```

```suggestion
  name: '${parTopLevelManagementGroupPrefix}-platform-${mg.key}${parTopLevelManagementGroupSuffix}'
```

Think we are missing the suffix additions